lib/documentLoader.js

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {didIo} = require('bedrock-did-io');
+const {documentLoader} = require('bedrock-jsonld-document-loader');
+
+module.exports = async url => {
+  let document;
+  if(url.startsWith('did:')) {
+    document = await didIo.get({did: url, forceConstruct: true});
```

Comment:
```suggestion
    document = await didIo.get({did: url});
```
The `forceConstruct` param has been removed (in the `did-veres-one#v13.x` branch that this PR is now using through `bedrock-did-io`).

Reply:
addressed here 9331932